docs/docs/guides/otel.md

```diff
@@ -208,3 +208,25 @@ of your application and apply the`KeptnAppContext`.
 Keptn will re-deploy your application and Jaeger should show a link to the previous trace in the references section.
 
 ![linked trace](./assets/linkedtrace.png)
+
+## OTEL Internal Data Processing
```

Comment:
Please remove this whole section, it's not necessary to explain, how the OTel collector works. People can read good documentation about it in the OTel docs that are linked already.

Reply:
@mowies in the DoD it's mentioned to 

1. Detailed explanation in the Otel guide page about how the otel collector wrks in case you do not want to use Jaeger allInOne.

Reply:
That is not needed. People can read about how the collector works in the OTel docs. No need to document this on our side.